image/v1/imagev1.go

```diff
@@ -120,6 +120,8 @@ func MakeV1ConfigFromConfig(img *image.Image, v1ID, parentV1ID string, throwaway
 	// Delete fields that didn't exist in old manifest
 	delete(configAsMap, "rootfs")
 	delete(configAsMap, "history")
+	delete(configAsMap, "os_version")
+	delete(configAsMap, "os_features")
```

Comment:
Now that we are starting to have more of these new fields, it may be a good idea to define a list of them at the top level, next to the `Image` struct. I'm impressed you updated this; I probably wouldn't have remembered to.